protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -39,6 +40,8 @@ const pathParamUniqueSuffixDeliminator = "_"
 
 const paragraphDeliminator = "\n\n"
 
+var nonAlphanumeric = regexp.MustCompile("[^a-zA-Z0-9 ]+")
```

Comment:
Could you add a comment to explain what this is used for? It seems a bit confusingly named to me, it seems to me to identify any combination of alphanumeric characters (and a space?).

Reply:
Does this behave as expected with non-ascii filenames?